vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -1083,7 +1083,7 @@ + (bool)isMediaURL:(NSURL *)url firstPartyURL:(NSURL *)firstPartyURL referrerURL
 
 - (NSString *)encodedURI:(NSString *)uri
 {
-  const auto encoded = ledger->URIEncode(std::string(uri.UTF8String));
+  const auto encoded = [self URIEncode:std::string(uri.UTF8String)];
```

Comment:
Entire method should be removed (And its declaration in BATBraveLedger.h) since the function no longer exists in `ledger.h`